Description: Eliminate dev-depedency on assert_matches
 The assert_matches crate is not packaged in Debian. Patch out
 the handful of tests that depend on it so the rest of the
 testsuite can run.
 
 In the longer term it looks assert_matches will become
 available as part of the standard library in stable
 rust, if this happens then we can switch over to that
 implementation.
Author: Peter Michael Green <plugwash@debian.org>

--- rust-rustyline-6.3.0.orig/Cargo.toml
+++ rust-rustyline-6.3.0/Cargo.toml
@@ -48,8 +48,6 @@ version = "1.0"
 
 [dependencies.unicode-width]
 version = "0.1"
-[dev-dependencies.assert_matches]
-version = "1.2"
 
 [dev-dependencies.doc-comment]
 version = "0.3"
--- rust-rustyline-6.3.0.orig/src/lib.rs
+++ rust-rustyline-6.3.0/src/lib.rs
@@ -995,9 +995,9 @@ impl<'a, H: Helper> Iterator for Iter<'a
     }
 }
 
-#[cfg(test)]
+/*#[cfg(test)]
 #[macro_use]
-extern crate assert_matches;
+extern crate assert_matches;*/
 #[cfg(test)]
 mod test;
 
--- rust-rustyline-6.3.0.orig/src/test/common.rs
+++ rust-rustyline-6.3.0/src/test/common.rs
@@ -173,11 +173,11 @@ fn newline_key() {
 
 #[test]
 fn eof_key() {
-    for mode in &[EditMode::Emacs, EditMode::Vi] {
+    /*for mode in &[EditMode::Emacs, EditMode::Vi] {
         let mut editor = init_editor(*mode, &[KeyPress::Ctrl('D')]);
         let err = editor.readline(">>");
         assert_matches!(err, Err(ReadlineError::Eof));
-    }
+    }*/
     assert_line(
         EditMode::Emacs,
         &[KeyPress::Char('a'), KeyPress::Ctrl('D'), KeyPress::Enter],
@@ -208,7 +208,7 @@ fn eof_key() {
     );
 }
 
-#[test]
+/*#[test]
 fn interrupt_key() {
     for mode in &[EditMode::Emacs, EditMode::Vi] {
         let mut editor = init_editor(*mode, &[KeyPress::Ctrl('C')]);
@@ -225,7 +225,7 @@ fn interrupt_key() {
             assert_matches!(err, Err(ReadlineError::Interrupted));
         }
     }
-}
+}*/
 
 #[test]
 fn delete_key() {
